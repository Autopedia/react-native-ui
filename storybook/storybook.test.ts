import initStoryshots from '@storybook/addon-storyshots';
jest.useFakeTimers();
jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');

initStoryshots({
  storyKindRegex: /^((?!.*VideoMessage.*).)*$/gm,
});
