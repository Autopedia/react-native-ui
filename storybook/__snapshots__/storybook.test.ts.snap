// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Button Default 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#D9D9D9",
        "borderRadius": 4,
        "borderWidth": 1,
        "flexDirection": "row",
        "opacity": 1,
        "paddingBottom": 10,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 10,
      }
    }
  >
    <Text>
      Hello Button
    </Text>
  </View>
</View>
`;
